docs/UserGuide.md

```diff
@@ -120,325 +127,416 @@ Displays the output of `View` command, which expands each ingredient or recipe c
+- After a [Find Ingredient](#445-find-ingredient) command, to see the full list of ingredients again, please use a
+[List Ingredient](#446-list-ingredient) command.
 
 **Format:**<br />
 `find ingredient <keyword>...`
 
 **Example(s):**<br />
-1. `find ingredient cream`
-   <br />Expected Output:<br />
-   ![findIngredient1.png](images/ingredientCommands/findIngredient1.png)
+1. Command: `find ingredient banana cream`
+<br />Expected Output:<br />
+![findIngredient1.png](images/ingredientCommands/findIngredient.png)
 
 **Additional Information:**<br />
-- Requirements for a keyword:
-    1. Keyword is case-insensitive.<br />
+- Keyword(s) are separated by spaces.
+  - e.g. "corn chicken beef" will be regarded as "corn", "chicken" and "beef". 
+Results will show matches for any of the 3 keywords.
+
+- Match Criteria:
+    1. Matching of keyword(s) is case-insensitive.<br />
        e.g. Finding with keyword: "corn" will match with "COrN"
-    2. Any keyword must be contained in the name of the recipe.<br />
-       e.g. Finding with keyword: "corn" will match with "CoRN flour", "coRn FlakeS", "popcorn" but not "`" <br />
-       e.g. Finding with keywords: "corn Chicken beef" will match "beef Chicken", "beef corn", etc. but not "beefcorn"
+    2. Ingredient(s) will be matched as long as a full keyword is in its name.<br />
+       e.g. Finding with keyword: "corn" will match with "cornflour" and "corn flakes" but not "apple core". <br />
 
-### 4.2.6 List Ingredients
+[\[Back to Table of Contents\]](#table-of-contents)
 
-List all the ingredients again after `find ingredient` Operation.
+#### 4.4.6 List Ingredient
+
+Switch to Ingredient [tab](#31-tabs) and list all ingredients.
 
 **Format:**<br />
 `list ingredient`
 
 **Example(s):**<br />
-1. `list ingredient`
+1. Command: `list ingredient`
 <br />Expected Output:<br />
-![listIngredient1.png](images/ingredientCommands/listIngredient1.png)
+![listIngredient1.png](images/ingredientCommands/listIngredient.png)
+
+[\[Back to Table of Contents\]](#table-of-contents)
 
-### 4.2.7 View Ingredients
+#### 4.4.7 View Ingredient
 
-View an ingredient in the [Main Window](#36-mainwindow).
+Open an ingredient in the [Main Window](#36-mainwindow).
 
 **Format:**<br />
 `view ingredient <index>`
 
 **Example(s):**<br />
-1. `view ingredient 3`
-   <br />Expected Output:<br />
-   ![viewCommand1.png](images/ingredientCommands/viewCommand1.png)
+1. Command: `view ingredient 3`
+<br />Expected Output:<br />
+![viewCommand1.png](images/ingredientCommands/viewIngredient.png)
 
 **Additional Information:**<br />
 - An index number is required for the Command. Refer to the indexes displayed for each [Card](#33-cards) in the
   [Side Bar](#32-side-bar).
 
-## 4.3 Recipes
-This sections covers commands related to RecipeBook management. Any command primarily interacting with recipes will
+[\[Back to Table of Contents\]](#table-of-contents)
+  
+### 4.5 Recipes
```

Comment:
Mention sorting of recipe ingredients on display somewhere..?(by name). I think recipes have no sorting.

Reply:
Recipe ingredients are not sorted I believe

Reply:
I don't think there's any form of sorting at at all for recipes?

Reply:
I think recipe ingredients is sorted tho? As in the order of display compared to the order of my -i tags

Yeah recipe is not sorted

Reply:
I think recipe ingredients is sorted tho? As in the order of display compared to the order of my -i tags

Yeah recipe is not sorted